lib/augmentConfig.js

```diff
@@ -370,4 +404,53 @@ function addProcessorFunctions(config) {
 	return config;
 }
 
-module.exports = { augmentConfigExtended, augmentConfigFull };
+/**
+ * @param {StylelintConfig} fullConfig
+ * @param {string} configDir
+ * @param {string} filePath
+ * @return {StylelintConfig}
+ */
+function applyOverrides(fullConfig, configDir, filePath) {
+	let { overrides, ...config } = fullConfig;
+
+	if (!overrides) {
+		return config;
+	}
+
+	if (!Array.isArray(overrides)) {
+		throw Error(
+			'The `overrides` configuration property should be an array, e.g. { "overrides": [{ "files": "*.css", "rules": {}}] }.',
```

Comment:
[suggestion] How about adding extra whitespace inside of the JSON object?

```suggestion
			'The `overrides` configuration property should be an array, e.g. { "overrides": [{ "files": "*.css", "rules": {} }] }.',
```